src/pages/home/report/ReportActionContextMenu.js

```diff
@@ -94,15 +97,15 @@ class ReportActionContextMenu extends React.Component {
             {
                 text: this.props.translate('reportActionContextMenu.copyLink'),
                 icon: LinkCopy,
-                shouldShow: false,
+                shouldShow: () => false,
                 onPress: () => {},
             },
 
             {
                 text: this.props.translate('reportActionContextMenu.markAsUnread'),
                 icon: Mail,
                 successIcon: Checkmark,
-                shouldShow: true,
+                shouldShow: () => true,
```

Comment:
NAB These don't need to be converted to a function if you use `_.result(contextAction.shouldShow)`

http://underscorejs.org/#result

Reply:
I thied this but it didn't seem to work, I think all the contextActions get mapped out in[ this line](https://github.com/Expensify/Expensify.cash/blob/bd3d3006c2c1e1bb33c4f846d208118575363b62/src/pages/home/report/ReportActionContextMenu.js#L187) when I did it.

Reply:
OK, I'm not quite sure what you tried, but just to be extra clear about what I'm proposing:

```suggestion
                shouldShow: true,
```

Then in the map needs changed to:
```
{this.contextActions.map(contextAction => _.result(contextAction.shouldShow) && (
```

If that's what you did, and it didn't work... I'd be curious to know why that didn't work?

Reply:
oh I managed to do it like this
`{this.contextActions.map(contextAction => _.result(contextAction, 'shouldShow', false) && (`

Reply:
ah, cool. Yeah, I guess I had the syntax a little bit off.